packages/rolldown/tests/fixtures/output/hashFileNames/_config.ts

```diff
@@ -28,33 +32,19 @@ export default defineTest({
     for (const chunk of renderChunks) {
       switch (chunk.facadeModuleId) {
         case path.join(__dirname, 'main.js'):
-          expect(chunk.fileName).toMatchInlineSnapshot(`"main-!~{000}~.js"`)
-          expect(chunk.imports).toMatchInlineSnapshot(`
-            [
-              "shared-!~{002}~.js",
-            ]
-          `)
-          expect(chunk.dynamicImports).toMatchInlineSnapshot(
-            `
-            [
-              "dynamic-!~{004}~.js",
-            ]
-          `,
-          )
+          expect(chunk.fileName).toMatch('main-!~{000}~.js')
```

Comment:
The trick not working here since it the switch branch is not mutually exclusive during the loop, so using `toMatch` instead, since it is static.